controller/two_lane_queue_test.go

```diff
@@ -55,14 +55,6 @@ func TestRateLimit(t *testing.T) {
 	}
 
 	q := newTwoLaneWorkQueue("live-in-the-limited-lane", rl)
-	// Verify the slow lane has the proper RL.
-	q.SlowLane().AddRateLimited("1")
```

Comment:
Looking a the controller.go code we never added things to the slow lane using AddRateLimited. The changes in this PR changed the slow queue type to a regular one (no rate limiting - since it's not needed)

I think this makes sense given if something is reconciled and fails it should be requeued on the fast lane

Reply:
Is it worth adding a comment to explain why we don't need rate limiting on the slow queue?  Is that because the consumer queue is limited, and we'll only add to the consumer queue from the slow queue when the fast queue is empty?